reactLogin/src/App.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import './App.css';
+import LoginForm from './components/LoginForm';
+import { useNavigate } from 'react-router-dom';
+
+const App: React.FC = () => {
+  const [message, setMessage] = useState<string>('');
+  const navigate = useNavigate();
+
+  const handleLoginSubmit = async (values: { username: string; password: string }) => {
```

Comment:
You can move this logic inside the loginform and there no need of the onsubmit anymore.